bundle/manifests/leader-election-rolebinding_rbac.authorization.k8s.io_v1_rolebinding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  creationTimestamp: null
+  name: leader-election-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: leader-election-role
+subjects:
+- kind: ServiceAccount
+  name: default
```

Comment:
I guess we want this to be a non-default service account too, don't we? @fedepaol

Reply:
yep, the whole point is to have the manager moved to another account. Also, how comes the olm lane passed if this is aiming at the wrong account? It might be that this permission is not really required?

Reply:
it's because the manager-account role already has all the permissions the leader election role has.
I am going to delete this role all at once.

Reply:
or even better, keep it, but bind to the new service account and remove those permissions from the account-role

Reply:
if we don't need it, there's no point in keeping it. CI seems to say we don't